src/config/theme.js

```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
   ...bodhiTheme.overrides,
   MuiButton: {
     root: {
+      fontSize: xSmallText,
+      lineHeight: lineHeight24,
+      padding: `4px ${px(space2X)}`,
```

Comment:
where did you get these padding values from?

Reply:
16px
![image](https://user-images.githubusercontent.com/18548885/48346814-407ae680-e6b7-11e8-87c9-bb792e994f11.png)
4px = (height 32px - line-height 24px) / 2